### PR TITLE
Decrease a time for making a workspace

### PR DESCRIPTION
**Pull request by eunwoosh**
### Summary
To decrease time for making a workspace, remove a code to reformat the python text using yapf.
This change makes python file saved in workspace a little unprettier, but it's still human readable.
How much time can be saved is as below:

- classification : -0.1 sec
- detection : -1.2 sec
- segmentation : -0.6 sec

Added code is copied from mmcv but remove yapf part.

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added e2e tests for validation.
- [x] I have added the description of my changes into CHANGELOG in my target branch (e.g., [CHANGELOG](https://github.com/openvinotoolkit/training_extensions/blob/develop/CHANGELOG.md) in develop).​
- [ ] I have updated the documentation in my target branch accordingly (e.g., [documentation](https://github.com/openvinotoolkit/training_extensions/tree/develop/docs) in develop).
- [ ] I have [linked related issues](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

### License

- [ ] I submit _my code changes_ under the same [Apache License](https://github.com/openvinotoolkit/training_extensions/blob/develop/LICENSE) that covers the project.
      Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```

### CODE REVIEW

File: otx/algorithms/common/adapters/mmcv/utils/config_utils.py [hunk 1]
```diff
@@ -161,7 +161,93 @@ def fromfile(filename, use_predefined_variables=True, import_custom_modules=True
         cfg_dict, cfg_text = MPAConfig._file2dict(filename, use_predefined_variables)
         if import_custom_modules and cfg_dict.get("custom_imports", None):
             import_modules_from_strings(**cfg_dict["custom_imports"])
-        return Config(cfg_dict, cfg_text=cfg_text, filename=filename)
+        return MPAConfig(cfg_dict, cfg_text=cfg_text, filename=filename)
+
+    @property
+    def pretty_text(self):
+        """Make python file human-readable."""
```
Comment (sungmanc) [inline otx/algorithms/common/adapters/mmcv/utils/config_utils.py]: If this function comes from the MMConfig, how about writing where it comes from?
```python
"""Make python file human-readable.""" --> """Modified version of MMConfig ~~~"""
```

**Review by sungmanc — DISMISSED** [review]
LGTM, with minor comment

File: otx/algorithms/common/adapters/mmcv/utils/config_utils.py (hunk 1 above)
Comment (eunwoosh) [reply otx/algorithms/common/adapters/mmcv/utils/config_utils.py]: That's a good idea. I'll update it.

**Review by goodsong81 — DISMISSED** [review]
A little bit concerned about copying code block, but let's keep going.
Better if we have a small test for the block.

**Review by jaegukhyun — APPROVED** [review]

**Review by goodsong81 — APPROVED** [review]

**Pull request merged**